Gemfile

```diff
@@ -75,6 +75,7 @@ group :production do
 end
 
 group :test do
+  gem 'simplecov', '~> 0.11.0', require: false
```

Comment:
> to have a controlled updated process in which one can bump versions as needed step by step thus reducing the propability of introducing errors due to the version changes

`Gemfile.lock` keeps our gems at the same version from machine to machine, and `bundle update <gem>` can be used to update gems individually without updating others.

> let me know if I should remove the version and I'll will update the PR accordingly ;)

Please do, thanks!